pyrs/dataobjects/fields.py

```diff
@@ -724,11 +726,12 @@ def coordinates(self) -> np.ndarray:
         return self._field.coordinates  # type: ignore
 
     @staticmethod  # noqa: C901
-    def __to_wksp_and_peaks(filename: str,
-                            peak_tag: str,
-                            projectfile: Optional[HidraProjectFile],
-                            hidraworkspace: Optional[HidraWorkspace],
-                            peak_collection: Optional[PeakCollection]) -> Tuple[HidraWorkspace, PeakCollection]:
+    def __to_pointlist_and_peaks(filename: str,
```

Comment:
why the double '_' ?

Reply:
private methods are marked with `__` [article](https://www.geeksforgeeks.org/private-methods-in-python/)